src/views/Player.vue

```diff
@@ -384,11 +419,14 @@ export default {
       }
     },
 
-    recordAnswer(plioQuestion, answer) {
+    recordAnswer(plioQuestion, answer, updatedProgressBarInfo) {
       // this function is called when the close button is clicked
       // Update state to "answered"
       plioQuestion["state"] = "answered";
 
+      this.progressBarInfo['completionPercent'] = updatedProgressBarInfo[
+        'updatedCompletionPercent']
+
```

Comment:
we're updating our current progress with the value that we received from `PlioQuestion.vue` after user has submitted and closed the question